lib/dangermattic/plugins/android_unit_test_pr_checker.rb

```diff
@@ -0,0 +1,128 @@
+# frozen_string_literal: true
+
+require_relative 'utils/git_utils'
+
+module Danger
+  # Plugin to detect classes without Unit Tests in a PR.
+  class AndroidUnitTestPRChecker < Plugin
```

Comment:
This was something I had in mind when I started with the prototype, to have some convention around PR checks to clearly separate them from Post Build checks that will use build artifacts such as coverage report, built files, test results, etc.
Now a bit deeper into the project, I don't think it makes a lot of sense. Hopefully the name of the plugin on its own will be enough to differentiate them; also, most of them will likely be PR checks, as you mentioned, and hopefully their names will be descriptive enough so that users of the Gem will be able to use a plugin whenever appropriate for them.

I can rename the plugin to something like `AndroidUnitTestChecker` or `AndroidMissingUnitTestChecker` or something like `AndroidNeedsUnitTests`. Wdyt?

Reply:
Rename the plugin to simply `AndroidUnitTestChecker`.